examples/pytorch/BertNewsClassification/MLproject

```diff
@@ -0,0 +1,22 @@
+name: bert-sentiment
+
+conda_env: conda.yaml
+
+entry_points:
+  main:
+    parameters:
+      epochs: {type: int, default: 5}
+      gpus: {type: int, default: 0}
+      batch_size: {type: int, default: 64}
+      num_workers: {type: int, default: 3}
+      learning_rate: {type: float, default: 1e-3}
+      tracking_uri: {type: str, default: 'http://localhost:5000'}
```

Comment:
Thanks @shrinath-suresh for the PR! Could we split out the examples into a separate PR? Also - I don't think we want to encourage passing the tracking URI as a parameter / defining a default value for the tracking URI within an MLproject file. The reason is that MLflow has its own default tracking URI, so by adding another tracking URI to the project we end up either duplicating the MLflow default or deviating from it (which may be confusing to users). We should instead encourage users to specify a custom tracking URI by setting the `MLFLOW_TRACKING_URI` environment variable

Reply:
done